_posts/2020-10-19-rust-2.0.0-release.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: post
+title: "Apache Arrow 2.0.0 (Rust)"
```

Comment:
Can you give this a more expressive title? It's going to look odd next to the general release post https://github.com/apache/arrow-site/pull/79/files#diff-4cab3ca69341d657c5c14187a3a6aed43c08417df10df8d07f34945ed801fe68R3

Reply:
Sure. I'm not feeling particularly creative right now but off the top of my head ...

- Apache Arrow 2.0.0 Rust Highlights
- Rust release notes for 2.0.0
- Highlights of Apache Arrow 2.0.0 Rust Implementation

If there are no opinions posted I will pick something later today.

Reply:
I like "Apache Arrow 2.0.0 Rust Highlights"